tests/functional/metrics.test.js

```diff
@@ -11,25 +11,21 @@ const fetchBrowsers = testDriver.Matcher.withFeature('fetchExt')
 
 const multipleApiCalls = asyncApiFns[1] // page should trigger 5 calls of 'setPageViewName'
 
-const init = {
-  metrics: { harvestTimeSeconds: 5 }
-}
-
 const loaderTypes = ['rum', 'full', 'spa']
 const loaderTypesMapped = { rum: 'lite', full: 'pro', spa: 'spa' }
 loaderTypes.forEach(lt => loaderTypeSupportabilityMetric(lt))
 
 function loaderTypeSupportabilityMetric (loaderType) {
   testDriver.test(`generic agent info is captured - ${loaderType}`, fetchBrowsers, function (t, browser, router) {
     let rumPromise = router.expectRum()
-    let metricsPromise = router.expectMetrics()
-    const loadPromise = browser.safeGet(router.assetURL('instrumented.html', {
-      loader: loaderType,
-      init
-    }))
-
-    Promise.all([metricsPromise, rumPromise, loadPromise])
-      .then(([{ request: data }]) => {
+    const loadPromise = browser.safeGet(router.assetURL('instrumented.html', { loader: loaderType }))
+
+    Promise.all([rumPromise, loadPromise])
+      .then(() => {
+        browser.get(router.assetURL('/'))
+        return router.expectMetrics(3000)
```

Comment:
These tests work great but i suspect that they would pass for metrics with an interval too (for instance if someone added it back in, the test would likely still pass).  

Should we add one more with a condition that it just checks that a payload isn't sent after x time? Or are we ok self-enforcing this one?

Reply:
Since took out any sort of timer in the source code, the only way to test it would be having one hang for some arbitrary amount of time, which only lengthens test suite duration. Prob not worth adding it as discussed. 🤷‍♂️